src/mame/includes/pdp11.h

```diff
@@ -0,0 +1,41 @@
+// license:GPL-2.0+
+// copyright-holders:SIMH project
```

Comment:
Does SIMH require copyright assignment and are they a legal entity?  If the answer to either is “no” you need to track down the actual copyright holders for this and fix it.  If that can’t be done, this should be re-implemented based on a description by someone who has not read the SIMH source.

(I’m uncomfortable with GPL capture for trivial things like this at the best of times.)

Reply:
Ah, something I remember.

Relevant: This might not be how we'd do things today, but we've a standing offer from Bob Supnik to reuse code from SIMH how we'd like in "MESS", I'll go highlight the email for you, Vas.

Reply:
Can we reuse SIMH code, then?   Parts of CPU core would be particularly interesting.